rtil/src/threads/ue/rebo/livesplit.rs

```diff
@@ -0,0 +1,44 @@
+use livesplit_core::{Segment, TimeSpan, TimingMethod};
+use crate::threads::ue::rebo::STATE;
+
+pub struct LiveSplit {
+    pub timer: livesplit_core::Timer,
+}
+pub struct Timer {}
+
+impl LiveSplit {
+    pub fn new() -> LiveSplit {
+        let mut run = livesplit_core::Run::new();
```

Comment:
Regular import works fine for `Run` as it doesn't clash with other names. Though for consistency sake with `LiveSplitTimer` it might make sense to import it as `LiveSplitRun`.